### DISCUSSION TITLE
Ranger/Ueberzug

### DISCUSSION THREAD

**Discussion by rebdeg** [hyprwm/Hyprland#656] [Q&A] [answered]
I use ranger, starts up with keybinding (kitty -e ranger). Works OK. However when ever I select an image, which will be opened by ueberzug, a new window is opened for that image. 
In other WM's ranger opens this viewer in the same terminal window. 
Is there en option to force ranger to open the viewer in that same window that it running in??

**Comment by fufexan** [comment, marked as answer]
Ueberzug works properly in Xorg only.

You said you use Kitty, which supports image display. I suggest using that instead, by putting these lines in `~/.config/ranger/rc.conf`
```sh
set preview_images true
set preview_images_method kitty
```

**Reply by rebdeg** [reply]
Thanks, that was a very easy fix!
